code/modules/xenoarcheaology/tools/tools.dm

```diff
@@ -310,3 +310,41 @@
 		usr << browse(null, "window=locater")
 
 	updateSelfDialog()
+
+/obj/item/device/xenoarch_multi_tool
+	name = "xenoarcheology multitool"
+	desc = "Has the features of the Alden-Saraspova counter, a measuring tape, and a depth analysis scanner all in one!"
+	icon_state = "ano_scanner2"
+	item_state = "lampgreen"
+	icon = 'icons/obj/xenoarchaeology.dmi'
+	origin_tech = list(TECH_MAGNET = 3, TECH_ENGINEERING = 3, TECH_BLUESPACE = 2)
+	matter = list(DEFAULT_WALL_MATERIAL = 10000,"glass" = 5000)
+	w_class = ITEMSIZE_SMALL
+	slot_flags = SLOT_BELT
+	var/scanning = 1
+	var/measuring = 0
+	var/obj/item/device/ano_scanner/anomaly_scanner = null
+	var/obj/item/device/depth_scanner/depth_scanner = null
+
+/obj/item/device/xenoarch_multi_tool/New()
+	anomaly_scanner = new/obj/item/device/ano_scanner(src)
+	depth_scanner = new/obj/item/device/depth_scanner(src)
+
+/obj/item/device/xenoarch_multi_tool/attack_self(var/mob/living/user)
+	depth_scanner.interact(user)
+
+/obj/item/device/xenoarch_multi_tool/verb/swap_settings(var/mob/living/user)
```

Comment:
These verbs are tested? I'm mostly concerned that they aren't in the Object tab, because that isn't explicitly declared here, and I don't write new verbs too often.

Reply:
Yep. They're usable via rightclicking on the item or typing the command in chat.